lib/base/variables.css

```diff
@@ -0,0 +1,68 @@
+.dark {
+  --bg_h: #1d2021;
+  --bg: #282828;
+  --bg_s: #32302f;
+  --bg1: #3c3836;
+  --bg2: #504945;
+  --bg3: #665c54;
+  --bg4: #7c6f64;
+
+  --fg: #fbf1c7;
+  --fg1: #ebdbb2;
+  --fg2: #d5c4a1;
+  --fg3: #bdae93;
+  --fg4: #a89984;
```

Comment:
use descriptive color names for reviewer to understand

Reply:
Those colors are for background(bg) and foreground(fg) with shades in numbers.
Hope you got it.
Thanks.